homeassistant/components/growatt_server/const.py

```diff
@@ -9,6 +9,7 @@
 
 SERVER_URLS = [
     "https://server-api.growatt.com/",
+    "https://server.growatt.com/",
```

Comment:
This is not the correct change to make, see also: 

https://github.com/home-assistant/core/pull/103858